foreman-proxy.te

```diff
@@ -153,6 +153,13 @@ ifdef(`distro_rhel6', `
     miscfiles_read_generic_certs(foreman_proxy_t)
 ')
 
+# Logrotate in RHEL7 does not allow signals, RHBZ#1527522
+gen_require(`
+    type logrotate_t;
+    type systemd_unit_file_t;
+')
+allow logrotate_t systemd_unit_file_t:service stop;
```

Comment:
I think it should only send signals which AFIAK is `kill`, but never stop. Is this part of `stop` in selinux?

Reply:
I am actually dropping this code and changing it to the macro that was introduced in RHEL 7.7.

Reply:
> I am actually dropping this code and changing it to the macro that was introduced in RHEL 7.7.

Is this macro available in EL < 7.7?